_posts/2016-06-30-servo-nightlies.md

```diff
@@ -0,0 +1,36 @@
+---
+layout:     post
+title:      Servo Nightly Builds Available
+date:       2016-06-30 20:00:00
+summary:    Try out the new Servo nightly builds
+categories:
+---
+
+Nightly builds of the Servo web rendering engine are now available. To make
+the Servo engine easy to interact with, we are bundling an HTML-based browser
+UI. While our engine is not yet fully web compatible, we want to give a larger
```

Comment:
I somewhat feel we should emphasize this more. Unsure how.

Also that some things (page load, for example) are still slow. I'm *very* apprehensive about expectations, especially since the press misconstrued that email in May. Perhaps I'm worrying too much :smile: